javascript-light-bulb.tar/javascript-light-bulb/main.js

```diff
@@ -0,0 +1,6 @@
+function changeStyle(){
+  const element=document.getElementById("container");
+  const element2=document.getElementById("button");
+  element.style.backgroundColor="beige";
+  element2.style.backgroundColor="yellow";
+}
```

Comment:
this is a good start, but think about using local variables outside of the function scope to keep track of whether or not the light bulb is `on` or `off`

The `changeStyle` function could check the value of a local variable to see if it's an `on` or `off` state, use an conditional to change your classes based on the value of the local variable, and set the local variable to the opposite state for the next time it's run.